content/04.workflows-and-software.md

```diff
@@ -42,141 +41,61 @@ These tools allow users to take advantage of the benefits of workflow software w
 privileges for software installation, thus enabling use by researchers working on shared cluster systems. However, even user-based software installation can encounter dependency conflicts. For example, you might need to use python2 to install and run a program (e.g. older scripts written by members of your lab), while also using snakemake to execute your workflows (requires python3.5). By installing each program into an isolated "environment" that contains only the software required to run that
 program, you can ensure all programs will run without issue. Using small, separate environments for your software and building many simple environments to accommodate different steps in your workflow also reduces the amount of time it takes conda to resolve dependency conflicts between different software tools ("solve" an environment). Conda virtual environments can be created and installed either on the command line, or via an environment YAML file, as shown. In this case, the environment file
-also specifies which Conda channels to search and download programs from. When specified in a YAML file, conda environments are easily transferable between computers and operating systems. Further, because the version of each package installed in an environment is recorded, workflow reproducibility is enhanced.](images/conda_figure.svg){#fig:conda_figure}
+also specifies which Conda channels to search and download programs from. When specified in a YAML file, conda environments are easily transferable between computers and operating systems. Further, because the version of each package installed in an environment is recorded, workflow reproducibility is enhanced. Although portions of conda may be superceded by alternative solutions [@url:https://github.com/QuantStack/mamba], this model of software installation and management will likely remain.](images/conda_figure.svg){#fig:conda_figure}
+
+#### Getting started with software management
+
+Using these software management tools within workflow systems is relatively straightforward: first install the manager, then add a system-specific line to each workflow step that specifies information for software installation. With snakemake, for example, each analysis step must contain a link to a software environment (e.g. on of the boxes described in **Figure {@fig:conda_figure}, B**). If the workflow is then executed with the "--use-conda" option, the software specified will be downloaded, installed, used to run that analysis step, and stored for future analysis runs. The specific syntax for using package managers with your chosen workflow system will be available in the documentation.
```

Comment:
It's not a requirement for each step to contain a link to a software environment in snakemake. 
You can also specify singularity from within snakemake, so if we're going to mention `--use-conda` specifically, it may be worth mentioning singularity as well or generalizing between the two.

Reply:
whoops, meant to say this was needed for snakemake + conda. But maybe I'll try to generalize the example.